user/contactmethod/contactmethod.go

```diff
@@ -8,12 +8,13 @@ import (
 
 // ContactMethod stores the information for contacting a user.
 type ContactMethod struct {
-	ID       string `json:"id"`
-	Name     string `json:"name"`
-	Type     Type   `json:"type"`
-	Value    string `json:"value"`
-	Disabled bool   `json:"disabled"`
-	UserID   string `json:"-"`
+	ID             string `json:"id"`
+	Name           string `json:"name"`
+	Type           Type   `json:"type"`
+	Value          string `json:"value"`
+	Disabled       bool   `json:"disabled"`
+	UserID         string `json:"-"`
+	FormattedValue string `json:"formattedValue"`
```

Comment:
Rather than adding a field here, add a `FormattedValue` method and then call it from graphql